include/ProcessTools.h

```diff
@@ -45,25 +45,93 @@
 
 #include "DataTypes.h"
 
-namespace Mezzanine
-{
-    namespace Testing
+namespace Mezzanine {
+namespace Testing {
+
+SAVE_WARNING_STATE
+SUPPRESS_CLANG_WARNING("-Wpadded")
+
+    /// @brief A simple struct for storing (some) output from a called process.
+    struct MEZZ_LIB CommandResult
     {
-        /// @brief Run a process and capture its console output.
-        /// @param Command The command to attempt to run and capture its output.
-        /// @param TempFileName The file to put all the console output of the command to be transferred back here.
-        /// @return Everything the subprocess emits to the console (stdout or stderr) as a single concatenated string.
-        /// @details This launches a process, redirects its output to a file and reads that file to return that output.
-        /// This is a simple wrapper around std::system, concatenating output redirection to the command. Because of
-        /// this "|", "<" and ">" are not allowed in commands passed to this.
-        Mezzanine::String MEZZ_LIB RunCommand(const Mezzanine::String& Command,
-                                              const Mezzanine::String& TempFileName);
-
-        /// @brief Get all the text in a file.
-        /// @param Filename The file to read.
-        /// @return A string with the contents.
-        Mezzanine::String MEZZ_LIB GetFileContents(const Mezzanine::String& Filename);
-    }// Testing
+        /// @brief The output to cout and cerr from the called process.
+        String ConsoleOutput;
```

Comment:
It is expected the stderr and stdout are merged in roughly chronological order in this?

Reply:
Stderr was a part of the initial implementation, but caused problems and was removed.  I'll update the docs so that only cout is mentioned.

Reply:
Would you be able to note that stderr is discarded and not available through this API?